site/scripts/db-indexes.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/first */
```

Comment:
My suggestion would be to use verb-like file names for scripts, like `scripts/do-something` or `scripts/do-something`. This will help us understand what each script does.